tests/installation/first_boot.pm

```diff
@@ -22,19 +22,8 @@ sub handle_login {
     if (get_var('DM_NEEDS_USERNAME')) {
         type_string $username;
     }
-    if (match_has_tag("sddm")) {
-        # make sure choose plasma5 session
-        assert_and_click "sddm-sessions-list";
```

Comment:
yeah, sddm can handle AUTOLOGIN already, and since we don't test NOAUTOLOGIN within KDE(even in upgrade cases) anymore, should be safely to remove.